javascript/javascript1/week3/Series.js

```diff
@@ -0,0 +1,50 @@
+const seriesDurations = [
+    
+    {
+      title: 'Friends',
+      days: 3,
+      hours: 14,
+      minutes: 32,
+    },
+    
+    {  
+      title: 'Suits',
+      days: 4,
+      hours: 2,
+      minutes: 0,
+         
+    },
+    {  
+        title: 'Castle',
+        days: 5,
+        hours: 9,
+        minutes: 48,
+             
+    },
+    {  
+        title: 'Big Little Lies',
+        days: 0,
+        hours: 11,
+        minutes: 40,
+                 
+    }
+];
+
+function calculateSeriesDuration(serDuration) {
+    let watchDuration = [];
+    let total = 0;
+
+    for (let i = 0 ; i < serDuration.length; i++) {
+      let hoursToDay = (serDuration[i].hours / 24 );  //Converting hours to days by dividing by 24
+      let day = ((hoursToDay + serDuration[i].days ) / 365 ) / 80 ;  //Calculating percentage of day in a  year divided by 80 (avg lifespan)
```

Comment:
confusing variable name: `day` actually does not contains days. `fractionOfLifeOnThisSeries` is a better variable name.